lib/irc/IrcEventBroker.js

```diff
@@ -347,6 +347,18 @@ IrcEventBroker.prototype.addHooks = function(client, connInst) {
             req, server, createUser(nick), chan, "join"
         ));
     });
+    this._hookIfClaimed(client, connInst, "nick", function(oldNick, newNick, chans, msg) {
+        chans = chans || [];
+        chans.forEach(function(chan) {
+            var req = createRequest();
```

Comment:
``var`` is evil, please use ``const`` here

Reply:
done